tools/gc_decode.cpp

```diff
@@ -24,15 +25,18 @@ void str_to_uint16(char *str, uint16_t *res, uint8_t base) {
 }
 
 void usage_error(char *name) {
-  std::cerr << "Usage: " << name << " [-raw] <global_code>" << std::endl
+  std::cerr << "Usage: " << name << " [-gc] <global_code>" << std::endl
             << "Usage: " << name << " -prontohex [-raw] <prontohex_code>"
-            << std::endl;
+            << std::endl
+            << "Usage: " << name << " -raw <freq> <raw_code>" << std::endl;
```

Comment:
You can remove `<freq>` as the decode routines don't use it. It's handled by the hardware IR receiving module. It's only use when **actually** sending a IR message. It won't matter in the _synthetic/simulated_ methods we are using in this program.